### PR TITLE
A11y - Updated input with no label.

### PR DESCRIPTION
**Pull request by ibrouos**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]: 
- [x] I have read and understand the pull request rules.

# Description
Updated search form input that is missing label. Added aria-label as no valid label in document. [Aria-label documentation](https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html)

Fixes #(issue)

## Type of change

Please delete any options that are not relevant.

- User interface (UI)
- Other (Accessibility)

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I ran ESLint and other linters for modified files
- [x] I have performed a self-review of my own code and tested it
- [x] I have commented my code, particularly in hard-to-understand areas
  (including JSDoc for methods)
- [x] My changes generate no new warnings
- [] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

Please do not use any external image service. Instead, just paste in or drag and drop the image here, and it will be uploaded automatically.

### CODE REVIEW

File: src/components/MonitorList.vue [hunk 1]
```diff
@@ -10,7 +10,7 @@
                     <font-awesome-icon icon="times" />
                 </a>
                 <form>
-                    <input v-model="searchText" class="form-control search-input" :placeholder="$t('Search...')" autocomplete="off" />
+                    <input v-model="searchText" class="form-control search-input" :placeholder="$t('Search...')" autocomplete="off" aria-label="Search monitored sites"/>
```
Comment (Computroniks) [inline src/components/MonitorList.vue:13]:
```suggestion
                    <input v-model="searchText" class="form-control search-input" :placeholder="$t('Search...')" autocomplete="off" aria-label="Search monitored sites" />
```
Could you fix the linting warning

**Review by Computroniks — CHANGES_REQUESTED** [review]

**Comment by louislam** [comment]
Should we use `$t('Search...')` instead? Because `Search monitored sites` is English only.

File: src/components/MonitorList.vue [hunk 2]
```diff
@@ -10,7 +10,7 @@
                     <font-awesome-icon icon="times" />
                 </a>
                 <form>
-                    <input v-model="searchText" class="form-control search-input" :placeholder="$t('Search...')" autocomplete="off" />
+                    <input v-model="searchText" class="form-control search-input" :placeholder="$t('Search...')" autocomplete="off" aria-label="Search monitored sites" />
```
Comment (CommanderStorm) [inline src/components/MonitorList.vue:13]: What do you think about such a change (plus adding this to the translation file(s) you can)

```suggestion
                    <input v-model="searchText" class="form-control search-input" :placeholder="$t('Search...')" autocomplete="off" :aria-label="$t('Search monitored sites')" />
```

Comment (zamentur) [reply src/components/MonitorList.vue:13]: On my side, i agree with this suggestion.

**Comment by CommanderStorm** [comment]
Closing this PR in favor of https://github.com/louislam/uptime-kuma/pull/4163, as I don't think the requested/suggested changes will be implemented

**Pull request closed without merging**